example/burrow_integration/account.py

```diff
@@ -0,0 +1,183 @@
+import os
+import binascii
+from iroha import IrohaCrypto
+from iroha import Iroha, IrohaGrpc
+from iroha.primitive_pb2 import can_set_my_account_detail
+import sys
+from Crypto.Hash import keccak
+
+if sys.version_info[0] < 3:
+    raise Exception("Python 3 or a more recent version is required.")
+
+# Here is the information about the environment and admin account information:
+IROHA_HOST_ADDR = os.getenv("IROHA_HOST_ADDR", "127.0.0.1")
+IROHA_PORT = os.getenv("IROHA_PORT", "50051")
+ADMIN_ACCOUNT_ID = os.getenv("ADMIN_ACCOUNT_ID", "admin@test")
+ADMIN_PRIVATE_KEY = os.getenv(
+    "ADMIN_PRIVATE_KEY",
+    "f101537e319568c765b2cc89698325604991dca57b9716b58016b253506cab70",
+)
+
+iroha = Iroha(ADMIN_ACCOUNT_ID)
+net = IrohaGrpc("{}:{}".format(IROHA_HOST_ADDR, IROHA_PORT))
+
+test_private_key = IrohaCrypto.private_key()
+test_public_key = IrohaCrypto.derive_public_key(test_private_key).decode("utf-8")
+
+
+def trace(func):
+    """
+    A decorator for tracing methods' begin/end execution points
+    """
+
+    def tracer(*args, **kwargs):
+        name = func.__name__
+        print('\tEntering "{}"'.format(name))
+        result = func(*args, **kwargs)
+        print('\tLeaving "{}"'.format(name))
+        return result
+
+    return tracer
+
+
+def make_number_hex_left_padded(number: str, width: int = 64):
+    number_hex = "{:x}".format(number)
+    return str(number_hex).zfill(width)
+
+
+def left_padded_address_of_param(
```

Comment:
The code looks quite good now, but the problem is that those functions are copy-paste in all files. I suggest to extract them into separate file, e.g. `integration_helpers.py` and import them from all files. Thanks to that all samples would be shorter